src/lib.jl

```diff
@@ -4844,11 +4844,11 @@ module Lib
     end
 
     function YMatrix_getIpointer(IvectorPtr)
-        ccall((:YMatrix_getIpointer, LIBRARY), Cvoid, (Ptr{Ptr{Cdouble}},), IvectorPtr)
+        ccall((:YMatrix_getIpointer, LIBRARY), Cvoid, (Ptr{Ptr{Complex{Float64}}},), IvectorPtr)
     end
 
     function YMatrix_getVpointer(VvectorPtr)
-        ccall((:YMatrix_getVpointer, LIBRARY), Cvoid, (Ptr{Ptr{Cdouble}},), VvectorPtr)
+        ccall((:YMatrix_getVpointer, LIBRARY), Cvoid, (Ptr{Ptr{Complex{Float64}}},), VvectorPtr)
```

Comment:
I don't think we should change this file. This is essentially auto-generated from the C header file using [`Clang.jl`](https://github.com/JuliaInterop/Clang.jl).

Reply:
Makes sense. Fixed...

Reply:
Awesome! Resolved in e831b89.